lib/harmony/harmonytask/doc.go

```diff
@@ -0,0 +1,47 @@
+/*
+	Package harmomnytask implements a pure (no task logic), distributed
+	task manager. This clean interface allows a task implementer to completely
+
+avoid being concerned with task scheduling and management.
+It's based on the idea of tasks as small units of work broken from other
+work by hardware, parallelizabilty, reliability, or any other reason.
+Workers will be Greedy: vaccuuming up their favorite jobs from a list.
```

Comment:
> Workers will be Greedy

This is not always the best strategy, tho it should result in a kinda-round-robin, so it's probably fine for now.

Reply:
Probably not worth doing in the first iteration, but we could build some more elaborate task claiming process where workers can say how much they want a task, and upsert their proposals for some short period.

Reply:
We do round robin after every accept. 
I propose that (for non-urgent work) we refuse acceping work in CanAccept() for a few polling periods if it's relatively undesirable (such as not having the storage locally yet it's local to another).